cmd/query/main.go

```diff
@@ -6,7 +6,10 @@ import (
 
 	"github.com/gin-gonic/gin"
 	"github.com/pborman/getopt/v2"
+	"github.com/swaggo/gin-swagger"
+	"github.com/swaggo/files"
 
+	_ "github.com/equinor/vds-slice/docs"
```

Comment:
I was hoping that we didn't need to check in the autogenerated swagger files (`/docs/*`), but that doesn't seam to be possible. Which is super annoying, as we need to keep running `swag init` every time we change some source docs

Reply:
What I don't like is that if we check it in, it easily gets out of sync. Let's say we check in docs/docs.go now. In my next pr I add a new endpoint with docstrings, but I forget to run `swag init`. Now `docs/docs.go` is out of sync

Reply:
But without that one who builds the code and has no interest in building docs is forced to install swag.

If it gets out of sync then it would not be a problem on server as it would be regenerated every time so users won't suffer.

We also can do something like create an empty docs/go file so that "install" doesn't complain, but with no actual content in it, and commit it? Not sure what will happen if we then add it to gitignore - will the local changes to it be ignored?
Or when testing we run swag init with output types which are not "go" and thus this file never changes?